kerlescan/profile_parser.py

```diff
@@ -94,8 +94,25 @@ def _parse_interface(name):
             "type", "N/A"
         )
 
+    def _parse_tags(tags):
+        """
+        helper method to convert tag lists to facts. We output a
+        fact for each unique tag (combination of namespace and key).
+        The value is a comma-separated ordered list of all values
+        for the tag.
+        """
+        tag_dict = {}
+        for tag in tags:
+            tag_name = "{}.{}".format(tag["namespace"], tag["key"])
```

Comment:
that is cool, I didnt know of this method before

Reply:
You mean the "{}".format() stuff?  It's pretty neat.  I think you can also do named substitution and pass in a dict with the values like this:
myvals = {"name": "eric", "host": "localhost"}
retval = "{name} some text {host}".format(myvals)
I'd check the docs first, however ;-)

Reply:
Just checked... not a dict, but a list of key=value.  And can do some formatting, too.